doc/manipulating_images/data_preparation.rst

```diff
@@ -144,15 +143,14 @@ Computing the mask
     It is also related to this example:
     :doc:`plot_nifti_simple.py <../auto_examples/04_manipulating_images/plot_nifti_simple>`.
 
-If a mask is not specified as an argument,
-:class:`NiftiMasker` will try to compute
-one from the provided neuroimaging data.
-It is *very important* to verify the quality of the generated mask by
-visualization. This allows to see whether it
-is suitable for your data and intended analyses.
-Alternatively, the mask computation parameters can still be modified. See the
-:class:`NiftiMasker` documentation for a complete list of mask computation
-parameters.
+
+If a mask is not specified as an argument, :class:`NiftiMasker` will try to
+compute one from the provided neuroimaging data.
```

Comment:
@banilo I think you should do your own PR instead of suggesting/askg changes on this one.